src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -723,6 +726,8 @@ export const selectAccountIdsByAssetIdAboveBalanceThresholdByFilter = createDeep
       : accountIdsAboveThreshold,
 )
 
+export type RowProps = Row<AccountRowData>
```

Comment:
Can we name this `AccountRowProps` and give it a better, less generic home?

Reply:
Done in [09cab7e](https://github.com/shapeshift/web/pull/10380/commits/09cab7eda0adfea64fc9a6d343482264710f73cc)